styles/alpha.h

```diff
@@ -38,4 +38,7 @@ class AlphaL {
   }
 };
 
+// To enable Gradient/Mixes constricted within Bump<> and SmoothStep<> layers
```

Comment:
Maybe add an example to make it clear how to use it?

Reply:
sure, added